packages/api/content/paths/html/001-common-elems/questions/story.md

```diff
@@ -0,0 +1,126 @@
+
+I know! We can add a title by using an HTML Header!
+
+The Header tag is used to represent the title of the page or content on the page.
+
+Let's give the content within our Paragraph tag a title: 
+
+><h1>Lorem Ipsum</h1>
+><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Nisl nunc mi ipsum faucibus vitae aliquet. Egestas sed sed risus pretium quam vulputate. Ut diam quam nulla porttitor. Turpis egestas pretium aenean pharetra magna ac. Aenean euismod elementum nisi quis eleifend quam adipiscing vitae. Id leo in vitae turpis massa sed elementum tempus. Cursus turpis massa tincidunt dui ut.</p>
+
+That looks great! Now we know that the paragraph is about Lorum Ipsum.
+
+So, how do we add a header? I'll show you!
+
+Let's examine the HTML code below:
+
+```html
+<h1>Lorem Ipsum</h1>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Nisl nunc mi ipsum faucibus vitae aliquet. Egestas sed sed risus pretium quam vulputate. Ut diam quam nulla porttitor. Turpis egestas pretium aenean pharetra magna ac. Aenean euismod elementum nisi quis eleifend quam adipiscing vitae. Id leo in vitae turpis massa sed elementum tempus. Cursus turpis massa tincidunt dui ut.</p>
+```
+
+You can see we added a Header `<h1></h1>` tag above our Paragraph tag. Any content within the opening and closing tags of the Header tag will be treated as a title.
+
+Usually the `<h1></h1>` tag is reserved for the main title of the page. As a rule of thumb, only one `<h1></h1>` tag should be used per page.
+
+If we want to emphasize other titles on our page, HTML gives us other options we can use. Let's take a look at how we can use different size titles through out our code:
+
+><h1>Lorem Ipsum</h1>
+><h2>Lorem Ipsum</h2>
+><h3>Lorem Ipsum</h3>
+><h4>Lorem Ipsum</h4>
+><h5>Lorem Ipsum</h5>
+><h6>Lorem Ipsum</h6>
+
+See how the title shrinks in the example above?
+
+We can use other Header tags to emphasize the main title of the page as well as titles of content on our page. Take a look at below at how we can do this:
+
+```html
+<h1>Lorem Ipsum</h1>
+<h2>Lorem Ipsum</h2>
+<h3>Lorem Ipsum</h3>
+<h4>Lorem Ipsum</h4>
+<h5>Lorem Ipsum</h5>
+<h6>Lorem Ipsum</h6>
+```
+
+# img Tag
+
+Our lesson on common HTML tags would not be complete without talking about images. Everyone loves to have some great quality images on their website, I know I sure do! I love to share all the creations I make at the bakery.
+
+Adding images to our website is simple! We use the Img `<img />` tag in our HTML to add images. 
+
+Wait, something looks a bit off! In our previous examples, we had an opening and closing tag but the Img Tag `<img />` only has one tag, is this right? Ofcourse it is! The Img `<img />` tag is a `self closing` tag. We can open and close the tag all within one tag by adding a `/` just before the `>` symbol in the opening tag.
```

Comment:
I think 'of course' is separate. Although I might be wrong since English is not my mother language. Lol!